src/components/PhotoList/PhotoList.js

```diff
@@ -11,18 +11,21 @@ function PhotoList({ photos }) {
   const location = useLocation();
 
   useEffect(() => {
-    if (location.pathname === '/') {
+    const pathSplit = location.pathname.split('/');
```

Comment:
const [,pathSplit] = location.pathname.split('/');

then you can use pathSplit like a normal variable

https://www.samanthaming.com/tidbits/13-skip-values-in-destructuring/